cli/azd/internal/repository/app_init.go

```diff
@@ -234,6 +234,27 @@ func prjConfigFromDetect(
 			}
 		}
 
+		if prj.HasWebUIFramework() {
+			// By default, use 'dist'. This is common for frameworks such as:
+			// - TypeScript
+			// - Vue.js
+			svc.OutputPath = "dist"
+
+		loop:
+			for _, dep := range prj.Dependencies {
+				switch dep {
+				case appdetect.JsReact:
```

Comment:
I believe that `build` is the output dir for CRA, right? For Vite, the default is `dist`. Can we account for that?

Reply:
That is indeed the current behavior. We default to `dist` for Vite.